packages/devtools/lib/src/timeline/flame_chart_canvas.dart

```diff
@@ -23,13 +23,9 @@ import 'timeline.dart';
 
 // TODO(kenzie): add tooltips to stack frames on hover.
 
-// TODO(kenzie): this should be removed once the cpu flame chart is optimized
-// and complete.
-const bool showCpuFlameChart = true;
-
 const _selectedFlameChartNodeColor = ThemedColor(
```

Comment:
you are using the light color for both options. Comment on why you are doing that.

Reply:
done.